Library/Homebrew/github_packages.rb

```diff
@@ -115,33 +120,42 @@ def upload_bottle(user, token, skopeo, formula_name, bottle_hash)
       ".#{rebuild}"
     end
     version_rebuild = "#{version}#{rebuild}"
-    root = Pathname("#{formula_name}-#{version_rebuild}")
+    root = Pathname("#{formula_name}--#{version_rebuild}")
```

Comment:
Why this naming?

Reply:
For consistency with the other files on disk in the same directory (bottles tarball and JSON):
```
wiggletools--1.2.10/
wiggletools--1.2.10.catalina.bottle.json
wiggletools--1.2.10.catalina.bottle.tar.gz
wiggletools--1.2.10.x86_64_linux.bottle.json
wiggletools--1.2.10.x86_64_linux.bottle.tar.gz
```

Reply:
@sjackman I think it'd be nicer to leave as-is but don't feel strongly either way if you do.

Reply:
I prefer the consistency for `ls` sort order and tab completion.